addons/mail/static/src/core/common/message_reactions.js

```diff
@@ -14,6 +15,7 @@ export class MessageReactions extends Component {
         this.store = useService("mail.store");
         this.ui = useService("ui");
         this.addRef = useRef("add");
+        this.isMobileOS = isMobileOS;
```

Comment:
It seems that elsewhere on the code it's usually called here, it probably avoids re-evaluating it at each rerender

```suggestion
        this.isMobileOS = isMobileOS();
```
(+ the opposite change in the template)